polls/models.py

```diff
@@ -12,13 +11,27 @@
 class Question(Page):
     subpage_types = ['polls.Choice']
 
+    def can_vote(user):
+        if PollVote.objects.filter(user=user) == []:
```

Comment:
This would mean a user can only vote once, not just for this poll, but for life.
Consider this check `return user.poll_votes.filter(question=self).exists()`
This uses the `related_name` on line 35 below to look at all of the current user's votes, and see if any of them were against the current Question